src/transformers/data/data_collator.py

```diff
@@ -446,7 +446,7 @@ def __call__(self, examples: List[Dict[str, torch.Tensor]]) -> Dict[str, torch.T
             "input_ids": input_ids,
             "attention_mask": self._tensorize_batch(attention_masks),
             "token_type_ids": self._tensorize_batch(segment_ids),
-            "masked_lm_labels": mlm_labels if self.mlm else None,
+            "labels": mlm_labels if self.mlm else None,
             "next_sentence_label": torch.tensor(nsp_labels),
         }
         if self.mlm:
```

Comment:
@gmihaila I also ran into this warning while doing BERT pretraining. I am wondering if we should remove lines 452 and 453 as they seem redundant.

Reply:
Yes you are perfectly right @vblagoje ! 452 and 453 are already coded inside of 449. I didn't look beyond that line 😄

Reply:
Ok, excellent @gmihaila . If you don't mind I can issue a PR. LMK

Reply:
I don't mind at all @vblagoje ! Thank you for noticing this! 😃